og.module

```diff
@@ -95,6 +95,28 @@ function og_entity_access(EntityInterface $entity, $operation, AccountInterface
 }
 
 /**
+ * Implements hook_entity_insert().
+ */
+function og_entity_insert(EntityInterface $entity) {
+  if (!\Drupal\og\Og::isGroup($entity->getEntityTypeId(), $entity->bundle())) {
+    return;
+  }
+
+  if (!empty($entity->hasField('uid'))) {
+    // Subscribe the group manager.
+    $membership = OgMembership::create(array('type' => OG_MEMBERSHIP_TYPE_DEFAULT));
+    $membership
+      ->setEntityId($entity->get('uid')->offsetGet(0)->getValue()['target_id'])
+      ->setEntityType('user')
+      ->setGid($entity->id())
+      ->setGroupType($entity->getEntityTypeId())
+      // todo: get the real name of the field.
+      ->setFieldName(OG_AUDIENCE_FIELD)
```

Comment:
lets add a function that gets the field name - for now you can hardcode `OG_AUDIENCE_FIELD` there.